iopipe/contrib/trace/auto_http.py

```diff
@@ -0,0 +1,131 @@
+import functools
+import uuid
+
+try:
+    from requests.sessions import Session
+except ImportError:
+    Session = None
+
+try:
+    from botocore.vendored.requests.sessions import Session as BotocoreSession
+except ImportError:
+    BotocoreSession = None
+
+from iopipe.compat import urlparse
+
+if Session is not None:
+    original_session_send = Session.send
+
+if BotocoreSession is not None:
+    original_botocore_session_send = BotocoreSession.send
+
+HEADER_BLACKLIST = ['Authorization']
```

Comment:
@ewindisch This currently will collect all headers other than `Authorization`. But there are other headers that could be sensitive in nature. Do you have an opinion on which HTTP headers we collect?

Reply:
This could be an interesting config setting - as for whatever reason (because they shouldn't contain sensitive data .. ) a user might not want to send `X-MyfancyNamespace-Thing`